### PR TITLE
Schema constants hierarchy stp

### PR DESCRIPTION
**Pull request by tyll**
This is a follow-up on #608

---
This change is [Reviewable](https://reviewable.io/reviews/nmstate/nmstate/612)

### CODE REVIEW

**Comment by tyll** [comment]
Also added the commits for OVSBridge.Port constants. They do not seem to be used in  libnmstate, therefore no changes were needed there.

File: libnmstate/schema.py [hunk 1]
```diff
@@ -297,3 +297,8 @@ class OVSBridge(object):
     PORT_NAME = 'name'
     PORT_VLAN_MODE = 'vlan-mode'
     PORT_ACCESS_TAG = 'access-tag'
+
+    class Port(object):
```
Comment (EdDev) [inline libnmstate/schema.py]: As I remember we discussed that the should use the linux-bridge vlan abstraction which we did a few weeks ago. See line 229. It needs to come with the schema change.
Lets do it in a separate PR.

Comment (tyll) [reply libnmstate/schema.py]: I do not recall the discussion and I am not sure what you mean. If you like, you can adjust this PR tomorrow and I can approve it. If the change for LinuxBridge.STP is ok, I can adjust this PR to only contain the respective commits.

Comment (EdDev) [reply libnmstate/schema.py]: Yes, the STP looks good, please extract the rest so we can take it. I mean, from this commit forward.
I will try to take over the OVS port/vlan change.

Comment (tyll) [reply libnmstate/schema.py]: removed the OVSBridge commits
